### ISSUE TITLE
Questions

### ISSUE THREAD

**Issue by pannet1** [swapniljariwala/zerodha#6]
The Challenge questions are standard IMHO. Only the answers are different. So in the usage example, are we supposed to replace the below questions with the actual questions.
q_arr = [
    'question1',
    'question2',
    "question3",
    'question4',
    'question5',
    'question6',
]

please provide with a exact instruction on how to run the sample file.

**Comment by swapniljariwala** [comment]
yes, you just need to copy paste (all possible) questions that can appear on "Security Questions" Page, 

When you register for the first time, you select a set of 5-6 questions, the same questions need to be added to array. The length of the q_arr does not matter, it should just cover all possible questions that you have chosen at time of registration, length of a_arr should be same as q_arr with answers in same sequence.

Hope that clarifies.

**Comment by pannet1** [comment]
Awesome swapnil,
transaction password has been removed. did you login recently with old python lib 3.7. can you please confirm if txn passwd is still required or not.

**Comment by swapniljariwala** [comment]
Not required. I will update the readme file. Thanks!

**Comment by pannet1** [comment]
hi swapnil,
thanks for the heads up. i stumbled upon [pyalgotrade](http://gbeced.github.io/pyalgotrade/docs/v0.18/html/index.html) libraray. i suspect you already are aware of it.
i am a novice user of python. i was able to run all the pyalgotrade example without error. i would love to see a similar experience with yours too.
what we need is data feed free of charge (at least what brokers receive) after logging in. i am told others (data providers) get many ticks per second at a huge cost from NSE. 
equipped with real-time data and our already tested strategy we just have to replace the trade functions of our strategy.

**Comment by Naveenkopalle** [comment]
HI Swapnil,

Im getting below error while trying to run the script.

Traceback (most recent call last):
  File "E:\Stocks\chrome\Kiteauth.py", line 21, in <module>
    h = k.orders()
  File "C:\Python27\lib\site-packages\kiteconnect\__init__.py", line 336, in orders
    return self._get("orders")
  File "C:\Python27\lib\site-packages\kiteconnect\__init__.py", line 455, in _get
    return self._request(route, "GET", params)
  File "build\bdist.win-amd64\egg\kitewrapper\__init__.py", line 206, in _request
KeyError: 'status'

anything i need to install in addition to your wrappers.

**Comment by rahulj175** [comment]
Hello Swapnil,

I am also getting the same error 
KeyError: 'status'

Let me know what needs to be done.

**Comment by rahulj175** [comment]
Following piece of code should follow the constructor of KiteFront & should be called before querying for holdings.

if k.connect():
    print('Logged in to Zerodha')
else:
    print('Check credentials')

P.S. README should be updated accordingly.

**Comment by shaantanu9** [comment]
sir i am new in coder i dont know coding and try to install this in my linux can you please guide me i know linux very well
